packages/suite/src/hooks/wallet/useUnstakeEthForm.ts

```diff
@@ -103,6 +110,25 @@ export const useUnstakeEthForm = ({
 
     const values = useWatch<UnstakeFormState>({ control });
 
+    useEffect(() => {
+        (async () => {
+            const { cryptoInput } = values;
+
+            if (!cryptoInput || Object.keys(formState.errors).length) {
+                setApproximatedEthAmount(null);
+
+                return;
+            }
+
+            const approximatedEthAmount = await Ethereum.simulateUnstake(
+                account.descriptor,
+                cryptoInput,
+            );
```

Comment:
This will not work because we do not allow desktop app to call unwhitelisted websites.

<img width="586" alt="Screenshot 2024-04-25 at 17 59 24" src="https://github.com/trezor/trezor-suite/assets/33235762/4930a4ab-6a20-4cfb-9006-c96be6e5763a">

We need to find solution to use Blockbook

Reply:
I'm waiting for you to add the feature to the Blockbook